install.ps1

```diff
@@ -0,0 +1,56 @@
+If (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(`
+    [Security.Principal.WindowsBuiltInRole] 'Administrator'))
+{
+    Write-Warning “You do not have Administrator rights to run this script!`nPlease re-run this script as an Administrator!”
```

Comment:
I got an error here, with the quotation character you have used: `“`. I changed that to `"` and it worked. Same with the quote close.